packages/circuits/README.md

```diff
@@ -81,4 +81,40 @@ A circuit to verify that a message in the email body, called command, is authori
 This is basically the same as the `email_auth.circom` described above except for the following features:
 - Instead of `subject_idx`, it additionally takes as a private input a padded email body `padded_cleaned_body` and an index of the command in the email body `command_idx`.
 - It extracts a substring `command` between a prefix `(<div id=3D\"[^\"]*zkemail[^\"]*\"[^>]*>)"` and a suffix `</div>` from `padded_cleaned_body`.
-- It outputs `masked_command` instead of `masked_subject`, which removes the invitation code with the prefix and one email address from `command`.
\ No newline at end of file
+- It outputs `masked_command` instead of `masked_subject`, which removes the invitation code with the prefix and one email address from `command`.
+
+## How to Build the Container Image for Unit Tests in GitHub Actions
```

Comment:
How often do we have to do this? Whenever we update tests?

Reply:
Basically, it is not frequent, but it is necessary to update when you want to change the version of circom or node